crates/astria-sequencer/src/transaction/hash.rs

```diff
@@ -0,0 +1,26 @@
+use anyhow::ensure;
+
+/// Represents the sha256 hash of an encoded transaction.
+pub(crate) struct Hash([u8; 32]);
```

Comment:
Please add methods on `Hash` instead of relying on the `TryFrom` impls, i.e.

```rust
fn from_bytes(&[u8]) -> Result<Self> {
}
```

Reply:
just gonna delete this file cause it's not actually used anymore